.pre-commit-config.yaml

```diff
@@ -0,0 +1,26 @@
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.5.0
+    hooks:
+    -   id: check-merge-conflict
+    -   id: debug-statements
+    -   id: mixed-line-ending
+-   repo: https://github.com/timothycrosley/isort.git
+    rev: 4.3.21-2
+    hooks:
+    -   id: isort
+        files: \.py$
+        # Following https://github.com/timothycrosley/isort/issues/694
+        args: [--multi-line=3, --trailing-comma, --force-grid-wrap=0, --use-parentheses,
+               --line-width=88, --project=tskit,_tskit,msprime]
```

Comment:
msprime is third-party here I think, strictly speaking.

Reply:
Yep, I thought so - however I'd seen it grouped with tskit in at one place so was seeing if that looked good throughout.